packages/blue-sdk-viem/contracts/vault-v2/GetVaultV2.sol

```diff
@@ -21,14 +22,20 @@ struct VaultV2Response {
     uint64 lastUpdate;
     address[] adapters;
     address liquidityAdapter;
+    bool isLiquidityAdapterKnown;
+    Caps liquidityCaps;
```

Comment:
This is not future-proof as the liquidity adapter could have several ids (eg. marketV1Adapter) and thus `liquidityCaps` should be a mapping and you should return the ids list (as you would need to compute them anyways)